docs/running-app.md

```diff
@@ -41,6 +41,28 @@ scripts/start-dev-local.sh
 npm run start:dev
 ```
 
+#### Debugging in Chrome Developer Tools
+TBD
```

Comment:
I added some notes to the prototype readme in case they're useful - https://github.com/ministryofjustice/hmpps-manage-recalls-prototype#to-debug-the-nodejs-app

Reply:
Linked.